src/main/python/SentimentAnalyzer.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+# pip install --upgrade azure-ai-textanalytics
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.core.credentials import AzureKeyCredential
+
+_key = "adc96f39ac2a4918a1f462b5bf8674b8"
+_endpoint = "https://nbatwittersentiment.cognitiveservices.azure.com/"
+
+def _authenticate(key:str,endpoint:str):
```

Comment:
Private methods use two underscores. I also thought it was one when I first started this project lol.

Reply:
private methods were refactored